src/main/java/io/codurance/pouch/Clock.java

```diff
@@ -0,0 +1,13 @@
+package io.codurance.pouch;
+
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+
+@Component
+public class Clock {
```

Comment:
Just use `Instant.now()` directly. No clock component needed.

Reply:
This component emerged while I was developing the service architecture in an outside-in TDD kind of way. My understanding was: I need this wrapper to be able to mock its behavior. I see the trade-off: I'm introducing small (lazy) classes that are poor in functionality, but I've seen others do it and have done it many times in Katas etc.

Reply:
I ask myself - does the design justify it's existence? Like to hear your (@SirXOPHER ) thoughts please

Reply:
@neilkidd-codurance also asked about my reasoning here. This is so I'm able to control the randomness of the `UUID` generator and to control the time given by `Instant` for testing.